.github/dependabot.yml

```
@@ -5,3 +5,11 @@ updates:
     schedule:
       interval: weekly
     open-pull-requests-limit: 10
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: weekly
+    open-pull-requests-limit: 10
+    ignore:
+      - dependency-name: "RDFLib"
+        versions: [">=6"]
```

I believe this is the correct way of ignoring verions as per this definition:
[versions—use to ignore specific versions or ranges of versions. If you want to define a range, use the standard pattern for the package manager (for example: ^1.0.0 for npm, or ~> 2.0 for Bundler).
](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore)

---

It is (we do this for other python repos). Though we usually use the yaml list syntax for versions:
```suggestion
        versions:
          - ">=6"
```